config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   root to: "home#show"
 
+  resources :sessions,  only: [:new]
```

Comment:
@njgheorghita we can change it another time, but I'm pretty sure `resources :sessions,  only: [:new]` and `get "/login", to: "sessions#new"` do the same thing.